Cohere.IntegrationTests/CohereStepDefinitions.cs

```diff
@@ -0,0 +1,32 @@
+using Reqnroll;
+using Xunit;
+
+namespace Cohere.IntegrationTests;
+
+[Binding]
+
+/// <summary>
+/// Defines the step definitions used by all Cohere integration tests
+/// </summary>
+public class CohereStepDefinitions
+{
+    public CohereClient _client;
```

Comment:
@AnjaliMohammed  -  based on the compiler warning, I would make the CohereClient here nullable, and add the required null checks